scripts/linters/js_ts_linter_test.py

```diff
@@ -209,39 +209,6 @@ def mock_compile_all_ts_files():
             ' file has a scope: {}.']
         self.validate(lint_task_report, expected_messages, 1)
 
-    def test_check_sorted_dependencies_with_unsorted_dependencies(self):
-        def mock_compile_all_ts_files():
-            cmd = (
-                './node_modules/typescript/bin/tsc -outDir %s -allowJS %s '
-                '-lib %s -noImplicitUseStrict %s -skipLibCheck '
-                '%s -target %s -typeRoots %s %s typings/*') % (
-                    js_ts_linter.COMPILED_TYPESCRIPT_TMP_PATH +
-                    'scripts/linters/test_files/', 'true', 'es2017,dom', 'true',
-                    'true', 'es5', './node_modules/@types',
-                    INVALID_SORTED_DEPENDENCIES_FILEPATH)
```

Comment:
Delete test files as well.

Reply:
This file has been used in other checks also. So I didn't delete it
example:- https://github.com/oppia/oppia/blob/2cab3f49e7921974c71ae7508432a611d1a8d49a/scripts/linters/js_ts_linter_test.py#L462

Reply:
Thanks for checking :)